Resources/Prototypes/_EE/Entities/Clothing/Neck/magmantles.yml

```
@@ -0,0 +1,39 @@
+          True: {state: icon-on}
+          False: {state: icon}
+
+- type: entity
+  id: ActionToggleMagmantle
+  name: Toggle Magmantle
+  description: Toggles the personal gravity field on and off.
+  categories: [ HideSpawnMenu ]
+  components:
+  - type: InstantAction
+    itemIconStyle: BigItem
+    event: !type:ToggleActionEvent
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)